scvi/model/_scanvi.py

```diff
@@ -75,45 +76,34 @@ def __init__(
         self,
         adata: AnnData,
         unlabeled_category: Union[str, int, float],
-        pretrained_model: Optional[SCVI] = None,
         n_hidden: int = 128,
         n_latent: int = 10,
         n_layers: int = 1,
         dropout_rate: float = 0.1,
         dispersion: Literal["gene", "gene-batch", "gene-label", "gene-cell"] = "gene",
         gene_likelihood: Literal["zinb", "nb", "poisson"] = "zinb",
         use_gpu: bool = True,
-        **model_kwargs,
+        encode_covariates: bool = False,
+        scanvae_model_kwargs: dict = {},
```

Comment:
```suggestion
        **model_kwargs,
```
and make sure to update docstring